Makefile

```diff
@@ -0,0 +1,41 @@
+WEBSITE_REPO=github.com/hashicorp/terraform-website
+BRANCH=kevin/terraform-plugin-mux-split
```

Comment:
Two things here:

- Can we update the `Makefile` variables `BRANCH`, `DOCKER_IMAGE`, `DOCKER_IMAGE_LOCAL`, and `DOCKER_RUN_FLAGS` to be prefixed with `WEBSITE_`? e.g. `WEBSITE_BRANCH` is much clearer 😄 
- Will this be pointed back to `master` on merge?

Reply:
This is leftover from local debugging; I'll fix it to `master`

Reply:
And Yes, I'll Prefix these vars to be clearer as well